src/Pos/Launcher/Runner.hs

```diff
@@ -220,6 +223,7 @@ runServer mkTransport mkReceiveDelay mkL (OutSpecs wouts) withNode afterNode oq
             mkListeners mkL' ourVerInfo theirVerInfo
     stdGen <- liftIO newStdGen
     logInfo $ sformat ("Our verInfo: "%build) ourVerInfo
+    notifyReady
```

Comment:
I think I would put this a bit later, so at least after we have opened our listening socket for example. You may need to talk to @avieth to find an appropriate place, and to do that in a way that's not too invasive.